### PR TITLE
Count badge

### PR DESCRIPTION
**Pull request by yurifrl**
I added a count badge that displays the count of the current window tabs.

### CODE REVIEW

**Comment by juni-vogt** [comment]
Hi, thanks a lot for your contribution! :)
I'll take a look at this next week, once my exams are over.

File: background.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const windowTabs = options => new Promise(function(resolve, reject) {
     currentWindow: true,
     pinned: false
   }, function(tabs) {
-    if (tabs.length >= options.maxWindow) {
```
Comment (juni-vogt) [inline background.js]: Oh wow, I did not see that :D

**Comment by juni-vogt** [comment]
Sorry for not getting back to this earlier. The changes are now live on the webstore!

**Comment by yurifrl** [comment]
Wow, great, thank you for the awesome software.

**Pull request merged**